packages/app/src/app/pages/Dashboard/utils/getChildCollections.js

```diff
@@ -0,0 +1,15 @@
+import getDirectChildren from './get-direct-children';
```

Comment:
It'd be best if this file would be renamed to `get-child-collections` for consistency.

Reply:
Changed and also changed getMostUsedTemplate to get-most-used-template